readme.md

```diff
@@ -85,7 +85,7 @@ controller.hears(['string','pattern .*',new RegExp('.*','i')],'message_received,
 
 ### Responding to Events
 
-Bots can respond to non-verbal events as well, like when a new user joins a channel, a file gets uploaded, or a button gets clicked. These events are handled using an event handling pattern that should look familiar. Most events in Botkit can be replied to like normal messages.
+Bots can respond to non-verbal events as well as when a new user joins a channel, a file gets uploaded, or a button gets clicked. These events are handled using an event handling pattern that should look familiar. Most events in Botkit can be replied to like normal messages.
```

Comment:
This change is not correct, there should be a comma after well.  Also I believe it is clearer to use "like" here.
```suggestion
 Bots can respond to non-verbal events as well, such as when a new user joins a channel, a file gets uploaded, or a button gets clicked. These events are handled using an event handling pattern that should look familiar. Most events in Botkit can be replied to like normal messages.
```